data/releases/schedule.yaml

```diff
@@ -11,6 +11,9 @@ schedules:
     - release: 1.26.1
       cherryPickDeadline: 2023-01-13
       targetDate: 2023-01-18
+    - release: 1.26.0
+      cherryPickDeadline: 2022-12-09
```

Comment:
Can we use something like `n/a` or `-` here? Looks like the `CherryPickDeadline` is just a string and does not necessarily have to be a date.

https://github.com/kubernetes/release/blob/e77e73e77719cedcd4d48567ee3c32c7e30fa7f8/cmd/schedule-builder/cmd/model.go#L36

Reply:
I was worried if that would break anyone who is potentially parsing that file automatically. Do we care about such cases in regard to this file?

Reply:
Remember that on the website side, we'll need to localize ”n/a” or whatever we use. I'd be happy to have the cherry pick deadline omitted in the source data for a minor release (and then verify at website build time that other patch releases do have this value available).

I don't think we've made any commitments about the format of that file as committed into [k/website](https://github.com/kubernetes/website/); in fact, I'm fairly sure we haven't.

Reply:
I discussed it with @saschagrunert and we decided to just provide an empty string for the cherry-pick deadline. Hugo seems to accept this without any issues. In this case, we don't need to change `schedule-bulider` to add `omitempty`.

Reply:
OK. We can iterate on that.

I might, separately, change the layout to turn an empty string into a localized “n/a” and to allow a null value there if we ever want to.